portfolio/src/main/webapp/script.js

```diff
@@ -79,4 +79,17 @@ function deleteComment(comment) {
   const params = new URLSearchParams();
   params.append('id', comment.id);
   fetch('/delete-comment', {method: 'POST', body: params});
+}
+
+function UserLogin(){
+   status = fetch('/user?loggedIn');
```

Comment:
`fetch` here would be an async ajax call, so you'll need to do the `fetch().then()` stuff.  Try to look at how last week you were sending a `fetch` request to get comments, and how you're reading the comments into the JS. It'll be the same here.

Reply:
I think I fixed this part.